connect/api/src/main/java/org/apache/kafka/connect/data/Struct.java

```diff
@@ -229,7 +229,12 @@ public void validate() {
             Object value = values[field.index()];
             if (value == null && (fieldSchema.isOptional() || fieldSchema.defaultValue() != null))
                 continue;
-            ConnectSchema.validateValue(fieldSchema, value);
+            try {
+                ConnectSchema.validateValue(fieldSchema, value);
+            } catch(DataException e) {
+                throw new DataException("Validate failed for required field: \""  + field.name() + "\", "
```

Comment:
Throwing new DataException(..., e) doesn't include original exception message here, it only set up the exception cause:
```
public Throwable(String message, Throwable cause) {
        fillInStackTrace();
        detailMessage = message;
        this.cause = cause;
    }
```
@lindong28